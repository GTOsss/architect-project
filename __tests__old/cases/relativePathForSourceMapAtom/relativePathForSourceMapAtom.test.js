const appRoot = process.cwd();
const { resolve } = require('path');
const { returnOutputPaths, promisifyCliCommand } = require('../../utils/utilsForArcStart');
const cleanOutPutBeforeTest = require('../../utils/cleanOutPutBeforeTest');
const { pathForCommand, casesOutputPath } = require('../casesConfigPath');

const currentCommand = pathForCommand.relativePathForSourceMapAtom;

describe('relativePathForSourceMapAtom', () => {
  cleanOutPutBeforeTest(casesOutputPath.relativePathForSourceMapAtom);

  test('test relative Path For SourceMapAtom', async () => {
    await promisifyCliCommand(`arc s -c ${currentCommand}`);
    const currentPathToOutput = resolve(appRoot, currentCommand, 'output');

    expect(returnOutputPaths(currentPathToOutput)).toMatchSnapshot();
  });
});
