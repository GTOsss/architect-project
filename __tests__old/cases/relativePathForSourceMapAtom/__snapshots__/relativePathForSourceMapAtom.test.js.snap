// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`relativePathForSourceMapAtom test relative Path For SourceMapAtom 1`] = `
Array [
  "{
  folder: 'src',
  files: [
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/clubs/clubs.module.scss',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/clubs/clubs.stories.tsx',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/clubs/clubs.tsx',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/clubs/index.ts',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/comands/comands.module.scss',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/comands/comands.stories.tsx',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/comands/comands.tsx',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/comands/index.ts',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/slesh/stadiums/index.ts',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/slesh/stadiums/stadiums.module.scss',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/slesh/stadiums/stadiums.stories.tsx',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/slesh/stadiums/stadiums.tsx',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/stadiums/index.ts',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/stadiums/stadiums.module.scss',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/stadiums/stadiums.stories.tsx',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/stadiums/stadiums.tsx',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/test/test2/comands/comands.module.scss',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/test/test2/comands/comands.stories.tsx',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/test/test2/comands/comands.tsx',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/test/test2/comands/index.ts',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/withoutSlesh/clubs/clubs.module.scss',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/withoutSlesh/clubs/clubs.stories.tsx',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/withoutSlesh/clubs/clubs.tsx',
    'C:/Users/mail/Desktop/timoha/arhitect/architect/__tests__/cases/relativePathForSourceMapAtom/output/src/page/withoutSlesh/clubs/index.ts'
  ]
}",
]
`;
